src/test/ValidateRedeemFei.sol

```diff
@@ -68,12 +89,15 @@ contract ValidationRedeemFei is Test {
         );
 
         AaveGovHelpers._passVote(vm, AAVE_WHALE, proposalId);
+        uint256 aDaiReserveBalanceAfter = IERC20(A_DAI).balanceOf(AAVE_MAINNET_RESERVE_FACTOR);
+        emit log_named_uint("Amount of FEI in LendingPool", aFeiPoolBalance);
+        emit log_named_uint("Max DAI Redeem from PSM after 3bps fee -----------", minBalance);
+        emit log_named_uint("PSM expected output amount from FEI in LendingPool", psmAmountOut);
+        emit log_named_uint("aDAI balance before ----------------", aDaiReserveBalanceBefore);
+        emit log_named_uint("aDAI balance after -----------------", aDaiReserveBalanceAfter);
+        emit log_named_uint("Difference between expected and real", aDaiReserveBalanceAfter - (aDaiReserveBalanceBefore + minBalance));
 
-        uint256 minBalance = aFeiAmount - (aFeiAmount * 3 / 10_000);
+        assertGe(aDaiReserveBalanceAfter, aDaiReserveBalanceBefore + minBalance);
```

Comment:
Can you comment the reason for using `assertGe` here. I.e that AAVE Mainnet Reserve Factor gets some additional aTokens minted to it while redeeming. 

And also link this as a comment: https://github.com/aave/protocol-v2/blob/baeb455fad42d3160d571bd8d3a795948b72dd85/contracts/protocol/libraries/logic/ReserveLogic.sol#L265-L325